src/app/pagenotfound/pagenotfound.component.scss

```diff
@@ -0,0 +1,44 @@
+@use '@angular/material' as mat;
+@import '/src/materialColorScheme.scss';
+
+.about-dockstore {
+  margin-top: 6rem;
+  background-image: url(../../assets/images/about-page/city.png);
+  background-repeat: repeat-x;
+  background-position-y: bottom;
+}
+
+.truck-img {
+  padding-bottom: 1rem;
+  height: 25%;
+  width: auto;
+}
+
+h3 {
+  font-size: 1.7rem;
+  color: mat.get-color-from-palette($dockstore-app-accent-1, darker);
+  line-height: 2.5rem;
+  font-weight: 500;
+}
+
+h2 {
+  font-weight: inherit;
+  font-size: 2.5rem;
+}
+
+hr {
+  width: 100%;
```

Comment:
Can use `w-100` instead of creating this class: https://github.com/dockstore/dockstore-ui2/blob/develop/src/bootstrap4.scss#L33